native/connector/virtmic.sh

```diff
@@ -62,10 +75,12 @@ tail -f "$fifoPath" | {
 
     # Read the new target node
     while read -r targetNodeSerial; do
+        UtilLog "[virtmic.sh] [Got FIFO Data] $targetNodeSerial"
         killMonitor
         disconnectInputs "$virtmicId"
-        setsid bash -- connect-and-monitor.sh "$virtmicPortFlId" "$virtmicPortFrId" "$targetNodeSerial" &
+        setsid bash -- connect-and-monitor.sh "$virtmicPortFlId" "$virtmicPortFrId" "$targetNodeSerial" 2>&1 >>`UtilGetLogPathForFile 'connect-and-monitor.sh'` &
```

Comment:
The order is wrong, it should be like this:

```suggestion
        setsid bash -- connect-and-monitor.sh "$virtmicPortFlId" "$virtmicPortFrId" "$targetNodeSerial" >>`UtilGetLogPathForFile 'connect-and-monitor.sh'` 2>&1 &
```

You can test this yourself by running this:

`(echo OUT; echo ERR 1>&2) 2>&1 1>>test_log.txt`
Prints ERR to stdout, OUT to test_log.txt

`(echo OUT; echo ERR 1>&2) 1>>test_log.txt 2>&1`
Prints OUT and ERR to test_log.txt

Reply:
We can also use:

```
exec 1>`UtilGetLogPathForFile $(basename $1)` 2>&1
```

At the start of each file (after sourcing utils.sh) instead of when running it.

Reply:
> We can also use:
> 
> ```
> exec 1>`UtilGetLogPathForFile $(basename $1)` 2>&1
> ```
> 
> At the start of each file (after sourcing utils.sh) instead of when running it. 

Shouldn't that be like this?
```
exec 1>>`UtilGetLogPathForFile $(basename $1)` 2>&1
```

Reply:
You are right